config/module.go

```diff
@@ -42,7 +42,11 @@ type Module struct {
 	Environment map[string]string `json:"env,omitempty"`
 
 	// FirstRunTimeout is the timeout duration for the first run script.
-	FirstRunTimeout time.Duration `json:"first_run_timeout,omitempty"`
+	//
+	// Setting this field to a zero value (e.g. 0 * [time.Second]) will // set the first run timeout to the default value of 1 hour, which
```

Comment:
```suggestion
	// Setting this field to a zero value (e.g. 0 * [time.Second]) will 
	// set the first run timeout to the default value of 1 hour, which
```
Lines seem off here.

Reply:
sorry, bad formatting - good catch